README.md

```diff
@@ -1,2 +1,10 @@
 # STM32Examples
 Provides several examples for the Arduino core for STM32 MCUs.
+For the complete description of each example, please refer to the comments at the beginning of each .ino file.
```

Comment:
I understand the rational but can't we have as well a short description with picture in the example forlder of examples/STM32L475VG-DISCOVERY-IOT ?